contracts/tokenLocking/TokenLockingDataTypes.sol

```diff
@@ -27,6 +27,10 @@ interface TokenLockingDataTypes {
   event UserTokenClaimed(address token, address user, uint256 amount);
   event UserTokenTransferred(address token, address user, address recipient, uint256 amount);
   event UserTokenWithdrawn(address token, address user, uint256 amount);
+  event UserTokenObligated(address token, address user, address obligatedBy, uint256 amount);
+  event UserTokenDeobligated(address token, address user, address obligatedBy, uint256 amount);
+  event UserTokenApproved(address token, address user, address approvedBy, uint256 amount);
+  event StakeTransferred(address token, address by, address from, address to, uint256 amount);
```

Comment:
I wonder if some of these values should be indexed?

Reply:
We only currently have the `TokenLocked` as an event with an indexed parameter. I don't recall a specific discussion, but I assume there was some logic here around these underlying events being triggered by colony / colonyNetwork, which would be emitting relevant events with indexed parameters?

Reply:
Ok, as long as we're happy about it.